fitbenchmarking/results_processing/performance_profiler.py

```diff
@@ -171,27 +175,20 @@ def create_plot(ax, step_values, keys):
 
     no_failures = np.zeros(len(step_values), dtype=np.int8)
```

Comment:
I think this can be removed as the elements are only used once, inside each iteration in order.

Reply:
Done